### ISSUE TITLE
NuGet Package Manager failed to load on experimental instance of VS

### ISSUE THREAD

**Issue by kartheekp-ms** [NuGet/Home#10877]
## Details about Problem

NuGet product used (NuGet.exe | Visual Studio | MSBuild.exe | dotnet.exe): VS 2022

Product version: 

Worked before? If so, with which NuGet version: 

## Repro steps and/or sample project

- Launch experimental instance of VS on `NuGet.Sln`
- Create a new SDK-style console application
- Navigate to Tools -> NuGet Package Manager (reported an error)

![image](https://user-images.githubusercontent.com/52756182/118708225-501b9e80-b7d0-11eb-9b42-ae4cc309b5a5.png)

After few seconds, VS reported an unhandled exception as shown below.

![image](https://user-images.githubusercontent.com/52756182/118708386-85c08780-b7d0-11eb-9c3b-ca45168e16f6.png)

Package Manager launched successfully on the regular instance of IDE but not on the experimental instance.

![image](https://user-images.githubusercontent.com/52756182/118708771-ff587580-b7d0-11eb-8cb2-78c10f912321.png)

**Comment by kartheekp-ms** [comment]
cc @jebriede

**Comment by donnie-msft** [comment]
@kartheekp-ms this is typical if your NuGet VSIX didn't install properly. Can you access other NuGet features in VS?

**Comment by kartheekp-ms** [comment]
@donnie-msft - I updated VS 2022 version today and here are my repro steps.

- Created a new VSIXProject (C# project)
- Opened NuGet PMUI on the solution successfully
- Launched VSIXProject or press F5
- Created an SDK-style console application (C#) in the experimental instance
- Tried to open PMUI in the experimentation instance which resulted in below exception

![image](https://user-images.githubusercontent.com/52756182/119390098-64a0e080-bc81-11eb-93cd-f4d0398ecbe2.png)

I closed the experimental instance and was able open NuGet PMUI on the solution successfully. I can open a VS feedback ticket incase if that helps.
